src/styles/index.scss

```diff
@@ -1,3 +1,231 @@
-body {
+// body {
+//   margin: 0;
+//   font-family: Roboto, sans-serif;
+// }
+
+// .card {
+//   max-width: 200px;
+// }
+
+// .card__container {
+//   // box-sizing: border-box;
+//   display: flex;
+//   flex-direction: column;
+//   border: 1px solid #f3f3f3;
+//   border-radius: 5px;
+//   max-width: 166px;
+//   margin: 0;
+//   align-items: center;
+//   padding: 32px 16px 16px;
+// }
+
+// .card__img {
+//   display: block;
+//   width: 160px;
+//   height: 134px;
+//   margin: 0;
+// }
+
+// .card__title {
+//   font-weight: 500;
+//   font-size: 12px;
+//   line-height: 18px;
+//   color: #060b35;
+
+//   margin-top: 40px;
+//   margin-bottom: 4px;
+// }
+
+// .card__desc {
+//   width: 100%;
+//   font-weight: 400;
+//   font-size: 10px;
+//   line-height: 14px;
+//   color: #616070;
+
+//   margin-bottom: 16px;
+// }
+
+// .card__review {
+//   width: 100%;
+//   display: flex;
+//   justify-content: space-between;
+
+//   margin-bottom: 24px;
+// }
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('./../images/star.svg');
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url('./../images/star-active.svg');
```

Comment:
Consider using SCSS variables for repeated values like colors or image paths to improve maintainability. For example, define a variable for the active star image path at the top of your SCSS file and use that variable within the selector.